src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -177,6 +186,44 @@ void scrollY(double deltaY) {
         orientation.scrollVertically(this, deltaY);
     }
 
+    void scrollXToPixel(double pixel) {
+        orientation.scrollHorizontallyToPixel(this, pixel);
+    }
+
+    void scrollYToPixel(double pixel) {
+        orientation.scrollVerticallyToPixel(this, pixel);
+    }
+
+    Val<Double> totalWidthEstimateProperty() {
+        return Val.wrap(orientation.widthEstimateProperty(this));
```

Comment:
Turns out `totalLengthEstimateProperty` and `totalBreadthEstimateProperty` already return `Val`s, so it would be better to change the return value of `orientation.widthEstimateProperty()` to `Val` than to do extra wrapping.